### PR TITLE
add: migration docs for Valkey

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
## Describe your changes
This pull request includes documentation for migrating external Redis and Valkey to Aiven for Valkey.

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

File: docs/products/valkey/howto/migrate-redis-valkey.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+Migrate your Redis®* databases, whether on-premise or cloud-hosted, to Aiven for Valkey, using Aiven Console's guided wizard.
+
+:::important
+Migrating from Google Cloud Memorystore for Redis is not currently
+supported.
+
+The version of the source Redis service cannot be higher than the version
+of the target Aiven for Caching service.
```
Comment (bogdanp05) [inline docs/products/valkey/howto/migrate-redis-valkey.md:15]: "the target Aiven for Valkey service"

File: docs/products/valkey/howto/migrate-valkey-aiven-for-valkey.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+Migrate your Valkey databases to Aiven for Valkey using the Aiven Console's guided wizard.
+
+## Prerequisites
+
+Before starting the migration process, ensure you have the following:
+
+- A target Aiven for Valkey service. To create one, see
+  [Get started with Aiven for Valkey](/docs/products/caching/get-started).
```
Comment (bogdanp05) [inline docs/products/valkey/howto/migrate-valkey-aiven-for-valkey.md:15]: the link should point to valkey

File: docs/products/valkey/howto/migrate-redis-valkey.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+:::
+
+## Prerequisites
+
+Before starting the migration process, ensure you have the following:
+
+- A target Aiven for Valkey service. To create one, see
+  [Get started with Aiven for Valkey](/docs/products/caching/get-started).
```
Comment (bogdanp05) [inline docs/products/valkey/howto/migrate-redis-valkey.md:23]: the link should point to valkey

**Review by bogdanp05 — CHANGES_REQUESTED** [review]

File: docs/products/valkey/howto/migrate-redis-valkey.md (hunk 3 above)
Comment (harshini-rangaswamy) [reply docs/products/valkey/howto/migrate-redis-valkey.md:23]: Thanks for catching these. I thought I had updated all.

**Review by bogdanp05 — APPROVED** [review]
Looks good. The plan is to merge when we go GA, right?

**Comment by harshini-rangaswamy** [comment]
> Looks good. The plan is to merge when we go GA, right?



> Looks g

**Pull request merged**